webrtc.html

```diff
@@ -399,23 +399,31 @@
 
         <ol>
           <li>
-            <p>Append an object representing the current call being handled
-            (i.e. function name and corresponding arguments) to the
-            <code>operations</code> array.</p>
+            <p>Let <var>p</var> be the result of transforming
```

Comment:
Heading aside, that's the tag document describing guidelines for using promises in specs, and is actively updated by @domenic - If we're going to follow a guideline here I believe this is the authoritative one. While this particular phrase may not be the clearest, isn't fixing that best done at the tag level, so everyone benefits and we're consistent?